lib/commands/check-coverage.js

```diff
@@ -6,6 +7,24 @@ exports.describe = 'check whether coverage is within thresholds provided'
 
 exports.builder = function (yargs) {
   yargs
+    .option('exclude', {
+      alias: 'x',
+      default: testExclude.defaultExclude,
+      describe: 'a list of specific files and directories that should be excluded from coverage, glob patterns are supported, node_modules is always excluded',
+      global: false
+    })
+    .option('exclude-after-remap', {
+      default: true,
+      type: 'boolean',
+      description: 'should exclude logic be performed after the source-map remaps filenames?',
+      global: false
+    })
+    .option('include', {
+      alias: 'n',
+      default: [],
```

Comment:
@coreyfarrell sorry for the post-review - shouldn't this be `['**']`?

Reply:
Nope.  An empty list is functionally equal to `['**']` but `[]` actually bypasses the check so it's more efficient.

https://github.com/istanbuljs/istanbuljs/blob/master/packages/test-exclude/index.js#L50-L54 is where the default is processed, `[]` is replaced with `false` which short-circuits the include check.